packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimePickerFooter.tsx

```diff
@@ -78,62 +78,81 @@ export const TimePickerFooter = (props: Props) => {
           variant="secondary"
           onClick={onToggleChangeTimeSettings}
           size="sm"
+          aria-expanded={isEditing}
+          aria-controls="time-settings-section"
+          icon={isEditing ? 'angle-up' : 'angle-down'}
         >
           <Trans i18nKey="time-picker.footer.change-settings-button">Change time settings</Trans>
         </Button>
       </section>
       {isEditing ? (
-        <div className={style.editContainer}>
+        <div className={style.editContainer} id="time-settings-section">
           <div>
```

Comment:
think this `<div>` is redundant - can we remove it?

Reply:
It is used for the `aria-controls` id. I could put it directly on the tabs maybe? 🤔